FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_Constructor.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using FrontDesk.Core.Aggregates;
+using PluralsightDdd.SharedKernel;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_Constructor
+  {
+    [Fact]
+    public void ConstructorCreatedAndIdAutoGenerated()
+    {
+      var scheduleId = Guid.NewGuid();
+      var range = new DateTimeRange(DateTime.Now, DateTime.Now.AddHours(3));
+      var appointment = new Appointment(1, scheduleId, 1, 1, 1, 1, range, "Title Test");
+
+      Assert.Null(appointment.DateTimeConfirmed);
+      Assert.Equal(1, appointment.AppointmentTypeId);
+      Assert.Equal(scheduleId, appointment.ScheduleId);
+      Assert.Equal(1, appointment.ClientId);
+      Assert.Equal(1, appointment.DoctorId);
+      Assert.Equal(1, appointment.PatientId);
+      Assert.Equal(1, appointment.RoomId);
+      Assert.Equal(3*60, appointment.TimeRange.DurationInMinutes());
+      Assert.Equal("Title Test", appointment.Title);
+    }
```

Comment:
In unit tests:
- Don't use the same value to mean two different things. So if ClientId is 1, make DoctorId 2 and PatientId 3, etc.
- Don't repeat literal values. Use `int testClientId = 1` and `int testDoctorId = 2` and then in your assertions use the variable not the literal value.